src/XenCert/StorageHandler.py

```diff
@@ -461,7 +463,10 @@ def DataPerformanceTests(self):
                         device_config['targetIQN'] = iqn
                         device_config['SCSIid'] = scsiId
                         sr_ref = self.session.xenapi.SR.create(util.get_localhost_uuid(self.session), device_config, '0', 'XenCertTestSR', '', 'lvmoiscsi', '',False, {})
-                        XenCertPrint("Created the SR %s using device_config: %s" % (sr_ref, device_config))
+                        device_config_with_hidden_password = device_config
```

Comment:
You should copy a new object

Reply:
I don't like so long name, just a suggestion you can make it short and add comment with details
```
# for hidden password in logging
device_config_tmp = ...
```

Reply:
Updated